src/ReactiveUI/ObservableAsPropertyHelper.cs

```diff
@@ -210,6 +210,27 @@ public static class OAPHCreationHelperMixin
             return ret;
         }
 
+        static ObservableAsPropertyHelper<TRet> observableToProperty<TObj, TRet>(
+        this TObj This,
+        IObservable<TRet> observable,
+        string property,
+        TRet initialValue = default(TRet),
+        bool deferSubscription = false,
+        IScheduler scheduler = null)
+    where TObj : IReactiveObject
+        {
+            Contract.Requires(This != null);
+            Contract.Requires(observable != null);
+            Contract.Requires(property != null);
+
+            var ret = new ObservableAsPropertyHelper<TRet>(observable,
```

Comment:
Can we, like, have each arg on its own line? And return directly instead of via `ret`?

Reply:
This is consistent with the non-nameof versions, do you want those changed too?

Reply:
No, as mentioned in Slack - let's just keep this focused on your changes. Otherwise we'll end up down a rabbit hole.